doc/autogen/types/vector.rst

```diff
@@ -26,6 +30,22 @@
     change the vector in any observable way but provides a hint about the
     size that will be needed.
 
+.. spicy:method:: vector::resize vector resize False void (n: uint<64>)
+
+    Resizes the vector to hold exactly *n* elements. If *n* is larger than
+    the current size, the new slots are filled with default values. If *n*
+    is smaller than the current size, the excessive elements are removed.
+
+.. spicy:method:: vector::sub vector sub False vector (begin: uint<64>, end: uint<64>)
```

Comment:
Since you are here documenting the sub() and at() methods, would this be a good time to also there document the begin() and end() methods, which have long-been overlooked, and omitted.

Reply:
Yeah, recorded in the documentation ticket already. I need to figure out how to get that into the autogenerated documentation (because it's a generic operator applied to different data types).